### PR TITLE
Include the network in L2 Sequencer Health logs

### PR DESCRIPTION
**Pull request by alecgard**
* Currently debugging can be challenging if an EA is checking the health for multiple networks, as you can't tell which network a log line is for. This commit adds a prefix to all logs to include the network, for easier debugging.

### CODE REVIEW

File: packages/sources/layer2-sequencer-health/src/adapter.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const makeExecute: ExecuteFactory<ExtendedConfig, endpoints.TInputParamet
   // Disclaimer on startup
   Object.keys(HEALTH_ENDPOINTS).forEach((network) => {
     if (!HEALTH_ENDPOINTS[network as Networks]?.endpoint)
-      Logger.info(`Health endpoint not available for network: ${network}`)
+      Logger.info(`[${network}] Health endpoint not available for network: ${network}`)
```
Comment (mmcallister-cll) [inline packages/sources/layer2-sequencer-health/src/adapter.ts:37]: we're always guaranteed to have a valid `network` here, right?

File: packages/sources/layer2-sequencer-health/src/network.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export const checkSequencerHealth: NetworkHealthCheck = async (
   })
   const isHealthy = !!Requester.getResult(response.data, HEALTH_ENDPOINTS[network]?.responsePath)
   Logger.info(
-    `Health endpoint for network ${network} returned a ${
+    `[${network}] Health endpoint for network ${network} returned a ${
```
Comment (mmcallister-cll) [inline packages/sources/layer2-sequencer-health/src/network.ts:52]: is there a difference with `[${network}]` vs `${network}` or is this just for formatting reasons?

**Review by mmcallister-cll — APPROVED** [review]

File: packages/sources/layer2-sequencer-health/src/adapter.ts (hunk 1 above)
Comment (alecgard) [reply packages/sources/layer2-sequencer-health/src/adapter.ts:37]: Yep! This uses `HEALTH_ENDPOINTS`, which is configured in the EA so we know it has networks. 

For the other logs, we use the input param `network` which is a required param.

File: packages/sources/layer2-sequencer-health/src/network.ts (hunk 2 above)
Comment (alecgard) [reply packages/sources/layer2-sequencer-health/src/network.ts:52]: Just formatting, figured the prefix `[...]` will make it easier to discern the different logs when grepping through.

**Pull request merged**